Assets/Scripts/GEGIntensityManager.cs

```diff
@@ -185,7 +191,7 @@ void RelaxM()
             EnemyPropertyChange("ZomBearHealth", 0, 100);
             EnemyPropertyChange("ZomBearSpeed", 0, 4);
             EnemyPropertyChange("ZomBearAttackDamage", 0, 10);
-            EnemyPropertyChange("ZomBearAttackRate", 0, (float)0.7);
+            EnemyPropertyChange("ZomBearAttackRate", 0, 0.7f);
         }
         void MediumM()
```

Comment:
RelaxM(), MediumM(), ... are hard to read. Rename to StartRelaxMode(), ... are better names.